src/main/java/no/nav/pto/veilarbfilter/auth/AuthUtils.java

```diff
@@ -15,10 +21,18 @@ public static VeilederId getInnloggetVeilederIdent() {
                 .orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "Id is missing from subject"));
     }
 
-    public static UUID getInnloggetVeilederUUID() {
-        return AuthContextHolderThreadLocal
-                .instance().getUid()
+    public static UUID getInnloggetVeilederUUID(AuthContextHolder authContextHolder) {
+        return authContextHolder.getIdTokenClaims()
+                .flatMap(claims -> getStringClaimOrEmpty(claims, "oid"))
                 .map(UUID::fromString)
-                .orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "Id is missing from subject"));
+                .orElse(null);
```

Comment:
Er det noen grunn til at det er endret til å returnere null isteden for throw exception her? :) (bare så ikke man endrer noen funksjonalitet som får konsekvenser et annet sted ) :P